### ISSUE TITLE
Wire up webpack and babelJS

### ISSUE THREAD

**Issue by isaacchansky** [codeforboston/project-hope#4]
We won't need this *immediately* but it would be good to get the plumbing for this set up earlier than later.

**Comment by rastapopolous** [comment]
Do you have a larger plan in mind for this?   Something react-oriented? cause it seems right now people are already writing code, in mostly whatever flavor of js is their preference...

**Comment by filipemir** [comment]
I wasn't around last night so someone correct me if I'm wrong, but as of the previous hack night we had discussed just generating a library of functions that would help us determine eligibility for the various programs. We can later use those in whatever client- or server-side application we come up with, but in the meantime we can run those in node or just in the console. So hopefully we can make choices for now that are agnostic about frameworks, etc.

Having said that, I agree that it would be good to agree at least on the flavor of JS we'll be using and have that enforced somehow. Having a basic setup to build the files into a coherent whole with Webpack and use them in node would be a good first step.

**Comment by isaacchansky** [comment]
Only enforcing the JS syntax (& future proofing for es6 in browsers) was the idea. Babel seems to me to be the standard these days for writing modern javascript and having it work basically everywhere, webpack being the latest thing to run tasks like babel. Talking about front end UI frameworks and stuff is out of scope for where my head was at in writing this issue. My goal was to just get some simple conventions set up as we start adding more JS modules so things don't go off the rails.
I think the only other thing we might consider at this point, to @filipemir point about enforcing the flavor of JS might be to add in something like ESLint to enforce some conventions as well, at the beginning. 
Certainly open to thoughts!

**Comment by rastapopolous** [comment]
Right, of course, that makes sense;  that way everything boils down to es5.  Sounds good.

**Comment by rastapopolous** [comment]
@filipemir re:
"generating a library of functions that would help us determine eligibility for the various programs"

assuming you mean a specific set of functions specific to each program that would walk through eligibility determination?

Just want to be sure we're all talking same general things... :)

**Comment by filipemir** [comment]
Yes, @rastapopolous. So whenever we have a React app or any such thing, we'll jsut import that logic:

```js
import  getMaHealthEligibility from 'awesome-eligibility-fns/state/ma/ma-health/get-eligibility';

// Then we use it however we need it
```

**Comment by filipemir** [comment]
I be at Hack night tonight, but Ill be happy to pick up this issue over the next week

**Comment by rastapopolous** [comment]
yup.  gotcha.
